frontend/src/Pages/ReleasePlan.jsx

```diff
@@ -33,8 +40,10 @@ const ReleasePlan = ({ projectId }) => {
           console.log(result);
           result.json().then((response) => {
             console.log(response);
-            setProblem(response.problemStatement);
-            setGoals(response.goalStatement);
+            console.log(response.fullySigned)
+            setFullySigned(response.fullySigned)
+            //setProblem(response.problemStatement);
+            //setGoals(response.goalStatement);
             setId(response.id);
           });
```

Comment:
Unnecessary setFullySigned call here. It will be set later on be fetchRelease.

Also, please clear your console logs and unneeded comments to remove bloat.